dev/VSIX/Extension/Directory.Build.targets

```diff
@@ -22,14 +22,6 @@
     <!-- Use 3-part Windows App SDK version, less any tagging suffix -->
     <VSIXVersion>$(WindowsAppSDKVersion)</VSIXVersion>
     <VSIXVersion Condition="$(VSIXVersion.Contains(&quot;-&quot;))">$(VSIXVersion.Substring(0, $(VSIXVersion.IndexOf("-"))))</VSIXVersion>
-    <!--
```

Comment:
Probably best to leave a comment about how this version is still required to be unique and increasing, and that this is now satisfied by the new scheme for `WindowsAppSDKVersion`.